src/ViewModel/SeeMoreLink.php

```diff
@@ -12,13 +12,14 @@ final class SeeMoreLink implements ViewModel
     use ArrayFromProperties;
 
     protected $name;
-
     protected $url;
+    protected $ariaLabel;
 
     public function __construct(Link $link, bool $isInline = false)
     {
         $this->name = $link['name'];
         $this->url = $link['url'];
+        $this->ariaLabel = $link['ariaLabel'];
```

Comment:
For `ariaLabel` we also need to check if it's set before assign it to `$this->ariaLabel`